docs/table_design/Data_distribution.md

```diff
@@ -571,14 +597,27 @@ Buckets reflect how data files are actually organized in StarRocks.
     ```
 
 - How to set the number of buckets when adding a new partition
-  > **NOTICE**
-  >
-  > You cannot modify the number of buckets for an existing partition.
-  - Method 1: automatically set the number of buckets (Recommended)
 
-    Since v2.5.7, StarRocks supports automatically setting the number of buckets based on machine resources and data volume for a partition. To enable this feature, make sure that the FE dynamic parameter `enable_auto_tablet_distribution` retains the default value `TRUE`.
+  - Method 1: automatically set the number of buckets (recommended)
```

Comment:
Line 601 to Line 620 are similar to the previous updates. See the comments on the previous updates.